community/modules/compute/gke-job-template/variables.tf

```diff
@@ -26,6 +26,12 @@ variable "node_count" {
   default     = 1
 }
 
+variable "completion_mode" {
+  description = "Sets value of `completionMode` on the job. Default uses indexed jobs. See [documentation](https://kubernetes.io/blog/2021/04/19/introducing-indexed-jobs/) for more information"
+  type        = string
+  default     = "Indexed"
```

Comment:
nit. Shall we add enum-validation `{Indexed, NonIndexed}` ?

Reply:
In the referenced docs it sounds like they are planning on expanding the list of options. I thought safer to leave it open so update will not be needed as options grow:

>_we are considering an IndexedAndUnique mode_